css/styles.css

```diff
@@ -0,0 +1,3 @@
+.book-groups{
```

Comment:
Hey @Joel-Nwamba ! Just a heads up that you probably don't want this in the main branch unless you want your `book-groups` to have a red border!

Reply:
My bad. I forgot to erase that.
It was just to make sure that the changes I made were working perfectly.

Reply:
Everything looks good on my end, and is updated. I don't think .book-groups would be necessary at all